<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccountDialog</class>
 <widget class="QDialog" name="AddAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddAccountDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="numberLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="initialBalanceLabel">
       <property name="text">
        <string>Initial Balance:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="initialBalanceDoubleSpinBox">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="warningBalanceLabel">
       <property name="text">
        <string>Warning Balance:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="warningBalanceDoubleSpinBox">
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="institutionLabel">
       <property name="text">
        <string>Institution:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="institutionComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="currencyLabel">
       <property name="text">
        <string>Currency:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="currencyComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>Owner:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QListView" name="ownersListView">
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="commentsLabel">
       <property name="text">
        <string>Comments:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="commentsTextEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="hideCheckBox">
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="includeInTotalCheckBox">
       <property name="text">
        <string>Include in Total</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
